readthedocs/core/static-src/core/js/sponsorship.js

```diff
@@ -1,29 +1,40 @@
 /* Read the Docs - Documentation promotions */
 
+var constants = require('./doc-embed/constants');
+
 module.exports = {
     Promo: Promo
 };
 
-function Promo (id, text, link, image) {
+function Promo (id, text, link, image, theme) {
     this.id = id;
     this.text = text;
     this.link = link;
     this.image = image;
+    this.theme = theme || constants.THEME_RTD;
     this.promo = null;
 }
 
 Promo.prototype.create = function () {
     var self = this,
-        nav_side = $('nav.wy-nav-side > div.wy-side-scroll');
-
-    if (nav_side.length) {
+        menu,
+        promo_class;
+    if (this.theme == constants.THEME_RTD) {
+        menu = this.get_sphinx_rtd_theme_menu();
+        promo_class = 'wy-menu';
+    }
+    else if (this.theme == constants.THEME_ALABASTER) {
+        menu = this.get_alabaster_menu();
+        promo_class = 'alabaster';
+    }
+    if (typeof(menu) != 'undefined') {
```

Comment:
Should we do something if we think we know what the theme is, but we can't find the menu?